cli/util/uploader.go

```diff
@@ -158,10 +163,11 @@ func uploadBin(ctx *cli.Context) error {
 					}
 					attrs := []object.Attribute{
 						*object.NewAttribute(attr, strconv.Itoa(int(blk.Index))),
-						*object.NewAttribute("primary", strconv.Itoa(int(blk.PrimaryIndex))),
-						*object.NewAttribute("hash", blk.Hash().StringLE()),
-						*object.NewAttribute("prevHash", blk.PrevHash.StringLE()),
-						*object.NewAttribute("timestamp", strconv.FormatUint(blk.Timestamp, 10)),
+						*object.NewAttribute("Primary", strconv.Itoa(int(blk.PrimaryIndex))),
+						*object.NewAttribute("Hash", blk.Hash().StringLE()),
+						*object.NewAttribute("PrevHash", blk.PrevHash.StringLE()),
+						*object.NewAttribute("Timestamp", strconv.FormatUint(blk.Timestamp, 10)),
+						*object.NewAttribute("Magic", magic),
```

Comment:
I thought we want this per-container, not per-object. Or you want to store multiple chains in a single container?